Testscripts/Windows/WORKLOAD-HB.ps1

```diff
@@ -5,41 +5,72 @@
 	Perform a simple VM hibernation in Azure
 	This feature might be available in kernel 5.7 or later. By the time,
 	customized kernel will be built.
-	# Hibernation will be supported in the general purpose VM with max 16G vRAM
-	# and the GPU VMs with max 112G vRAM.
+	Hibernation will be supported in the general purpose VM with max 16G vRAM
+	and the GPU VMs with max 112G vRAM.
 
 .Description
 	1. Prepare swap space for hibernation
 	2. Compile a new kernel (optional)
 	3. Update the grup.cfg with resume=UUID=xxxx where is from blkid swap disk
-	4. Run the first fio testing
+	4. Run the first storage or network workload testing
 	5. Hibernate the VM, and verify the VM status
 	5. Resume the VM and verify the VM status.
 	6. Verify no kernel panic or call trace
-	7. Run the second fio testing.
+	7. Run the second storage or network workload testing.
```

Comment:
I'm not very familiar with hibernation. I'm just confused that if VM has been resumed, the fio process which is started before hibernation would be unfrozen?

Reply:
After waking up, all running processes are back from the hibernating image in the disk. This test does not cover the live process resume, but validate the after-hibernate VM can continue invoking the same network or storage workload.